src/daemons/BenchmarkDaemon.cpp

```diff
@@ -0,0 +1,120 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
```

Comment:
Let's move it into tools dir,  and use another name "SimpleKVVerifyTool"

Reply:
Should I move it as test case or benchmark ?

Reply:
No,  it is not a UT or a benchmark